packages/react/src/Header/HeaderNarrowScreenOnlyGridCell.tsx

```diff
@@ -0,0 +1,13 @@
+import clsx from 'clsx'
+import { ForwardedRef, forwardRef } from 'react'
+import { Grid, GridCellProps } from '../Grid'
+
+export const HeaderNarrowScreenOnlyGridCell = forwardRef(
```

Comment:
`HeaderGridCellForNarrowWindow`? (or `In` instead of `For`)

(We describe the distinction between a screen and a window in the Grid docs.)

Reply:
It isn't necessarily 'for' or 'in', it only shows on narrow windows... 

`HeaderGridCellNarrowWindowOnly`?